### PR TITLE
feat: 🎸 new progress circle component for angular

### PR DESCRIPTION
**Pull request by sjkayle**
New Progress Circle component for Green Angular:

image

### CODE REVIEW

File: libs/angular/src/lib/progress-circle/progress-circle.component.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    constructor() {}
+  
+    calculateDegrees(percent: number): number {
+      if (percent > 100) {
+        return 180;
+      } else if (percent < 0) {
+        return 0;
+      }
```
Comment (eweseong) [inline libs/angular/src/lib/progress-circle/progress-circle.component.ts:29]: let's break it into multiple `if` block for simpler structure

```
if (percent > 100) {
  return 180;
}

if (percent < 0) {
  return 0;
}
```

**Review by eweseong — APPROVED** [review]

**Comment by sjkayle** [comment]
Hello @vsjolander. We are planning to use this component in a few of our applications in P&I team.
Here are some screenshots from our Figma:

image

image

**Comment by vsjolander** [comment]
Ok @sjkayle, thanks! Who's the designer?

I think this component will be used by more so need to get it documented in Design Library and get into Figma a reusable component! 😄

**Comment by sjkayle** [comment]
@vsjolander It was designed by Boon Ying of CX/Spartans team from KL.

**Pull request merged**